lib/board.rb

```diff
@@ -61,7 +61,14 @@ def valid_coordinates_suite(ship)
 
   def valid_placement?(ship, coordinates)
     @coordinates = coordinates
-    valid_coordinates_suite(ship) && ship_overlap(ship, coordinates)
+    @ship = ship
+    valid_coordinates_suite(ship) && ship_overlap(ship, @coordinates)
```

Comment:
Why aren't these @ship? I guess I am curious about why we needed to line 54 (I think I see you have another helper method below, will check there now). Love figuring out your thought process! Code review is getting funner every day!!!

Reply:
See line 69 comment for resolve.-